### PR TITLE
Add sracat.

### PR DESCRIPTION
**Pull request by wwood**
Add sracat, a program for extracting .sra files faster.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by luizirber** [comment]
Hah, I was about to open a PR and saw this one just in time =P
https://github.com/bioconda/bioconda-recipes/compare/master...luizirber:sracat

File: recipes/sracat/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  url: https://github.com/lanl/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - ncbi-ngs-sdk
```
Comment (luizirber) [inline recipes/sracat/meta.yaml:22]: [This](https://github.com/sourmash-bio/wort/pull/41/files#diff-720de93c99b71895fb83beb6b48e1264a3e7a1e72f2f94d7dd0e62603a16a34bR6-R22) is what I did for building it in a docker container, so I think it is missing `sra-tools` here

File: recipes/sracat/build.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+make \
+    SRA_PATH=${PREFIX} \
```
Comment (luizirber) [inline recipes/sracat/build.sh:2]: Should this be
```suggestion
    SRA_PATH=${BUILD_PREFIX} \
```
instead?

**Comment by wwood** [comment]
Hi @luizirber thanks for the comments. I couldn't get this to work building locally after trying some other things. I suspect the best course of action would just be to PR your branch? 

I don't think sra-tools is required, but could be wrong. I can build sracat outside docker without sra-tools installed.

**Pull request closed without merging**